website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,178 @@
+# Overview - Single API version operator
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
```

Comment:
is not it specific for go projects only, why it is in the main menu? should not it be in the golang section ?

Reply:
@camilamacedo86 This guide should only be for migrating Go operators. Ansible/Helm might have their own depending on what we actually change for those types.
 
So for now the location doesn't matter as long as this doc isn't shown on the website. We can put this doc in `website/content/en/docs/golang/kubebuilder-migration` but I think we still don't want to expose this on the website just yet while the other integration work is not yet complete.

Reply:
hI @hasbro17, 

This is the point. It is now in  `website/content/en/docs/kubebuilder-migration/project_migration_guide.md ` when I think that it should be in the golang dir.

Reply:
@camilamacedo86 These docs shouldn't be visible no matter where we place them. And we're already adding the new docs for Kubeubilder CLI at `website/content/en/docs/kubebuilder`:
https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs/kubebuilder 

To avoid mixing these docs before the integration is complete I want to keep it separate from `website/content/en/docs/golang` for now. 
Once the integration is finished with all the docs and we're ready to make them visible on the website we can move them to that location.
Same for ansible/helm docs whenever we add those in.

@varshaprasad96 Can you please move this doc to `website/content/en/docs/kubebuilder/migration/project_migration_guide.md`.

I'll move `website/content/en/docs/kubebuilder` to `website/content/en/docs/golang/kubebuilder` in a follow up PR.